services/payment/src/app.js

```diff
@@ -0,0 +1,55 @@
+import express from "express";
+import logger from "pino-http";
+import pinoPretty from "pino-pretty";
+import swaggerUi from "swagger-ui-express";
+import YAML from "yamljs";
```

Comment:
See one of the other services for using `swagger-jsdoc`

Reply:
OK